src/assets/images/backdrop.svg

```diff
@@ -0,0 +1,553 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
Check that you can use this svg instead: https://backdrop.so/_next/image?url=%2Fimages%2Fbackdrop-light.svg&w=128&q=100

Reply:
Sure!